velero-plugins/common/types.go

```diff
@@ -1,37 +1,31 @@
 package common
-
 type routingConfig struct {
-	Subdomain string `json:"subdomain"`
+      Subdomain string `json:"subdomain"`
 }
-
 type imagePolicyConfig struct {
-	InternalRegistryHostname string `json:"internalRegistryHostname"`
+      InternalRegistryHostname string `json:"internalRegistryHostname"`
 }
-
 // APIServerConfig stores configuration information about the current cluster
 type APIServerConfig struct {
-	ImagePolicyConfig imagePolicyConfig `json:"imagePolicyConfig"`
-	RoutingConfig     routingConfig     `json:"routingConfig"`
+      ImagePolicyConfig imagePolicyConfig `json:"imagePolicyConfig"`
+      RoutingConfig     routingConfig     `json:"routingConfig"`
 }
-
 const BackupServerVersion string = "openshift.io/backup-server-version"
 const RestoreServerVersion string = "openshift.io/restore-server-version"
-
```

Comment:
I'm fine with removing some line breaks but would prefer if we were grouping the constants and breaking by lines.